diagnostic.md

```diff
@@ -40,31 +47,39 @@ like to show all movies favorited by a profile on
 `http://localhost:3000/profiles/1`
 
   ```md
-    # < Your Response Here >
+    The purpose of the serializer is to make the view outputted in the way
+    one wants.
   ```
 
   ```rb
   class ProfileSerializer < ActiveModel::Serializer
```

Comment:
```ruby
class ProfileSerializer < ActiveModel::Serializer
  attributes :id, :given_name, :surname, :email
  has_many :movies
end
```

Use `has_many :movies` since we might want more than just the movie `id`s.